clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -635,6 +634,14 @@ private void maybeUpdateLastSeenEpochIfNewer(final Map<TopicPartition, OffsetAnd
         });
     }
 
+    public Map<TopicPartition, OffsetAndMetadata> allConsumed() {
+        return allConsumed;
+    }
+
+    public void allConsumed(Map<TopicPartition, OffsetAndMetadata> allConsumed) {
+        this.allConsumed = allConsumed;
+    }
+
```

Comment:
We should probably stick to the convention of using `set` for setters. Also, it would be prudent to make the given map immutable so that when other code invokes the getter they're not able to modify it. 

```suggestion
    public Map<TopicPartition, OffsetAndMetadata> latestPartitionOffsets() {
        return latestPartitionOffsets;
    }

    public void setLatestPartitionOffsets(Map<TopicPartition, OffsetAndMetadata> offsets) {
        this.latestPartitionOffsets = Collections.unmodifiableMap(offsets);
    }
```